CONTRIBUTING.md

```diff
@@ -71,6 +71,8 @@ The following sections describe how to manage dependencies in practice.
 
 ### package-lock.json
 
+Note: The `package-lock.json` files are only relevant when you execute an `npm install` in _this repository_. Users of `@instana` npm packages will have their own `package-lock.json` or `yarn.lock` file for their application, those are completely independent from the `package-lock.json` files in this repository. Thus, they need to take care of updating their transitive dependencies themselves. We only can make sure that the version ranges we use for our direct dependencies (in our `package.json` files) allow to install a dependency tree that has no known vulnerabilities.
```

Comment:
This was previously in the section "Updating A Single Version In A Lockfile", but it makes more sense here.